VCFv4.3.tex

```diff
@@ -1427,7 +1427,9 @@ \subsection{Header}
 
 The BCF2 header begins with the ``BCF2 magic'' 5 bytes that encode {\tt BCF\em XY} where {\em X} and {\em Y} are bytes indicating the major number (currently 2) and the minor number (currently 2).
 This magic can be used to quickly examine the file to determine that it's a BCF2 file.
-Immediately following the BCF2 magic is the standard VCF header lines in text format, beginning with \verb|##fileformat=VCFvX.Y|.
+Immediately following the BCF2 magic is an unsigned 4-byte integer containing the length of the header text which includes the NUL terminator character.
+The next portion of the file is a standard VCF header in text format with a NUL terminator.
+The header must begin with a VCF version number in the format \verb|##fileformat=VCFvX.Y|.
```

Comment:
This is better, but a bit oddly grouped: ‘immediately following the BCF2 magic is the length; the next portion is the text VCF header’ — but it's length/text-header/NUL that's semantically a unit (with three subunits).

“The header must begin”: ‘header’ here is ambiguous (BCF2 header or VCF text header?) and I'm not so keen on using ‘must’ here. We already use MUST language for `##fileformat` in §1.4.1, so IMHO this is clearer as a clause elucidating the previous sentence — somewhat thus:

> a standard VCF header in text format, from `##fileformat…` to `#CHROM…` inclusive, terminated by a NUL character.

(and similarly in the other two documents)